lib/get-latest-ticket-numbers.rb

```diff
@@ -11,8 +11,9 @@
 # year-364 days means we will delete 1 xtra day and be compliant for 23:59:59h
 # but then we will run again in 24h time
 
-today = Date.today.prev_day
-lastyear = today - 365
+today = Date.today
+yesterday = Date.today.prev_day
```

Comment:
Is this used anywhere besides the following line? If not, consider merging this line and the next.

Reply:
```ruby
lastyear = today.prev_day - 365
```
or
```ruby
lastyear = today - 366
```

Reply:
@blairboy362 also refactored similar lines in all other active scripts. Very useful comment, thanks!